chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/InheritanceErrorsReportCreatorJob.java

```diff
@@ -168,7 +168,7 @@ private InheritanceErrorsReportDTO check(CertifiedProductSearchDetails listing)
                 if (!productDetailsUrl.endsWith("/")) {
                     productDetailsUrl += "/";
                 }
-                productDetailsUrl += "#/product/" + listing.getId();
+                productDetailsUrl += env.getProperty("listingDetailsUrl").substring(1) + listing.getId();
```

Comment:
This just feels a little weird. In the above "if" (which is existing code), it is making sure the URL ends in a "/" and then the line you changed to use the property is ignoring the first "/" in the listingDetailsUrl. I know we don't want to have two "//"s but we do have complete control over the values in the properties file so it seems unnecessary to have to do these checks. Maybe the values aren't standardized in the way that they should be.. I don't know! I won't hold up the PR for this comment if changing it would require too many other changes but if you could take a look at it to see if this is all really needed I would appreciate it.